frontend/tests/specs/landing-page.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { expect, test, LandingPage } from "../fixtures/page-fixtures";
+import AxeBuilder from "@axe-core/playwright";
+
+test.describe("Landing Page", () => {
+  // Initialize page before each test, wait for the landing splash to be visible
```

Comment:
Minor nit on this, and no stress, but we try to do complete sentences for comments if they're on their own line - so capped as you have it and with a period. If it's inline, then no cap and no period :) I'll get these 😊